drivers/sensor/ti_hdc2xxx/Kconfig

```diff
@@ -0,0 +1,11 @@
+# TI_HDC2XXX temperature and humidity sensor configuration options
+
+# Copyright (c) 2021 Aurelien Jarno
+# SPDX-License-Identifier: Apache-2.0
+
+config TI_HDC2XXX
+	bool "Texas Instruments HDC2XXX Temperature and Humidity Sensor"
+	depends on I2C
+	help
+	  Enable driver for TI HDC2XXXX temperature and humidity sensors
```

Comment:
I see an extra 'X' here. Isn't it?

Reply:
Moreover shouldn't it be HDC20XX? Or are you planning to have also 21XX 22XX and so on?

Reply:
Good catch, there is indeed an extra X, I'll fix that. I have no idea if TI has planned a HDC21XX or HDC22XX. Given the HDC1XXX series actually has only see chips in HDC10XX, it might be safe to assume the same for the HDC20XX series. I'll fix that.

Reply:
I have just pushed a new version that rename the driver to HDC20XX instead.

Reply:
This sounds resolved - @avisconti - do you have any other concerns?